src/Core/Application/Service/TaskPool.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Service;
+
+class TaskPool
```

Comment:
Please introduce an interface and use it as a dependency, because it uses across the layers

Reply:
Fixed